### PR TITLE
Small UI fixes

### PR DESCRIPTION
**Pull request by kailamjeter**
## Description

Addresses: 

- Searching on Population / Violations data layers takes you back to water systems layer when zooming to a result
- Bug where you have to click twice on data layer to see a layer change on nationwide map

## Testing and Reviewing

Tested locally + in sandbox: https://kailajeter.leadout-sandbox.blueconduit.com/

Clicked around nationwide map, switched layers when zoomed into address, searched new address in violations layer.

### CODE REVIEW

File: client/src/model/slices/map_data_slice.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ const mapSlice = createSlice({
     },
     mapDataCleared: () => {
       return {
-        data: {},
+        mapData: { dataLayers: Array.from(ALL_DATA_LAYERS.keys()) },
```
Comment (justinebreuch) [inline client/src/model/slices/map_data_slice.ts:45]: Good call

Comment (kailamjeter) [reply client/src/model/slices/map_data_slice.ts:45]: Yeah was a sneaky bug...

**Review by justinebreuch — APPROVED** [review]

**Comment by beekley** [comment]
(I'm merging this while Kaila is OOO)

**Comment by kailamjeter** [comment]
Thanks for merging this!

**Pull request merged**